README.md

```diff
@@ -148,13 +73,14 @@ journalctl -fu docker.service # Journalctl/Systemd
       docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest VDVS_LOG_LEVEL=debug
       ```
 
-**ESX Plugin logs**
+**vDVS Driver logs**
 
 * Log location: `/var/log/vmware/vmdk_ops.log`
 * Config file location: `/etc/vmware/vmdkops/log_config.json`  See Python
 logging config format for content details.
 * **Turning on debug logging**: replace all 'INFO' with 'DEBUG' in config file, restart the service
 
+Detailed information regarding vDVS configuration is found at [vDVS configuration page](http://vmware.github.io/docker-volume-vsphere/documentation/configuration.html).
```

Comment:
Rewording suggestion : Please refer [vDVS configuration page](http://vmware.github.io/docker-volume-vsphere/documentation/configuration.html) for detailed steps.

Reply:
done!